### ISSUE TITLE
Getting warnings Unable to resolve chart-element & chart-attribute

### ISSUE THREAD

**Issue by przemwo** [grofit/aurelia-chart#28]
Hi, I'm using Aurelia 1.0. I'm getting these warnings when trying to start my app:

```
Unable to resolve aurelia-chart/aurelia-ugchart/elements/chart-element
Unable to resolve aurelia-chart/aurelia-ugchart/attributes/chart-attribute
```

The problem shows up when working with version 0.2.6.
When I switch to 0.2.5 there is no warnings and everything seems to be working fine.
How can I get rid of these warnings with 0.2.6?

**Comment by grofit** [comment]
Almost every single error people have with this plugin is the loading it, some people have it working fine in JSPM, CLI, Webpack, others dont. Due to this there are changes made to attempt to satisfy this but then it breaks for someone else. However with all this being said you seem to be using aurelia-ugchart which was a fork someone took a while back to try and get it working in JSPM so you may want to ping them.

**Comment by przemwo** [comment]
Hi, thank you for the answer.
Looks like I found the root of the problem though.
You reference to `aurelia-ugchart` in your package.json file:
https://github.com/grofit/aurelia-chart/blob/master/package.json#L86-L87
When I changed it to `aurelia-chart` everything seem to be working fine.

**Comment by grofit** [comment]
have published 0.2.7 based upon your findings hope this solves your problem, if so can you close the issue, if not feel free to leave it open and add more info.

**Comment by przemwo** [comment]
Yes, this solves the problem. Thank you very much!
